src/components/LearnerActivityTable/data/utils.js

```diff
@@ -0,0 +1,43 @@
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+
+export const trackDataTableEvent = ({
+  shouldTrackRef,
+  enterpriseId,
+  eventName,
+  tableId,
+  options,
+}) => {
+  if (shouldTrackRef.current) {
+    // track event only after original API query to avoid sending event on initial page load
+    // only track event when user performs manual data operation (e.g., pagination, sort, filter)
+    sendEnterpriseTrackEvent(
+      enterpriseId,
+      eventName,
+      {
+        tableId,
+        ...options,
+      },
+    );
+  } else {
+    // set to true to enable tracking events on future API queries
+    // eslint-disable-next-line no-param-reassign
+    shouldTrackRef.current = true;
+  }
+
+  return shouldTrackRef.current;
+};
+
+export const updateUrlWithPageNumber = (pageNumber, location, navigate, replace = true) => {
```

Comment:
[nit] Similar to previous comment, create a seperate unit test with a preference for integration test to create a reproducible test across all Datatable components using this function

Reply:
`trackDataTableEvent` and `updateUrlWithPageNumber` are only being called (without using their return values) within the hook and component, respectively. Therefore, I will write a separate test for them in the utils.test.js file.